README.md

```diff
@@ -17,37 +17,77 @@
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
 
-# 📖 [Set-up-mobile-first-portfolio] <a name="about-project"></a>
+# 📖 [Nescafe Basement Show] <a name="about-project"></a>
 
-**[Set-up-mobile-first-portfolio]** is a project in which we use our knowledge about html and css and we will add linters to make sure the code is clean.
+**[Nescafe Basement Show Website]** Nescafe Basement is a Pakistani music television series and platform that showcases emerging and established musical talent from the country. The show features live performances and covers of both and original songs across various genres, and aims to provide a platform for artists to showcase their skill and gain wider recognition.
 
-## 🛠 Built With <a name="built-with"></a>
+## 🛠 Built With HTML CSS and Javscript <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-I used VS code to create the project and files needed plus used Git and GitHub to complete the tasks. I used templates as well as flexbox for layout plus css grid layout. For mobile menu we have used javascript to give html some dynamic features.
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://en.wikipedia.org/wiki/HTML">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="">N/A</a></li>
+  </ul>
+</details>
 
 ### Key Features <a name="key-features"></a>
 
-- **[It will show my portfolio website made by using css and html.]**
-
-- **[For Mobile : And when you will click on menu button the menu will appear]**
+- **It shows html css and javascript skills of developer.**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-https://bilalrajput09.github.io/My-Portfolio/
+- [Live Demo](https://bilalrajput09.github.io/first_capstone_project_concert_webpage/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🚀 Screenshot <a name="screenshot"></a>
+
+![image](./assets/project%20screenshot.png)
+
+## 💻 Loom Presentation Video <a name="loom-video"></a>
+
+- [Loom Link](https://www.loom.com/share/bbd9761db353491d9a966b74a281e55d)
```

Comment:
- [ ] [OPTIONAL] The video provided here is recorded by another person for another project. I highly recommend you record video documentation as suggested in the project requirements. 🤝️

**Project requirements**
![image](https://user-images.githubusercontent.com/17243325/224772132-e6d523f8-e546-4fe5-b7f0-e0c5b5721ddd.png)